packages/react-instantsearch-dom/src/index.js

```diff
@@ -65,3 +65,6 @@ export { default as QueryRuleCustomData } from './widgets/QueryRuleCustomData';
 
 // Utils
 export { createClassNames } from './core/utils';
+
+// voiceSearchHelper
+export { default as createVoiceSearchHelper } from './lib/voiceSearchHelper';
```

Comment:
Unlike other flavours, in React-IS, this helper is directly used inside the VoiceSearch component. So we need to export this for users to create their own component.